packages/perseus/src/widgets/cs-program/cs-program.test.ts

```diff
@@ -37,5 +39,34 @@ describe("cs-program widget", () => {
         expect(container).toMatchSnapshot("first mobile render");
     });
 
-    // This widget doesn't have any direct behavior, it just renders an iframe
+    it("should show default user input before user interaction", () => {
+        const apiOptions = {
+            isMobile: false,
+        } as const;
+
+        const {renderer} = renderQuestion(question1, apiOptions);
+        const userInput =
+            renderer.getUserInput()[0] as PerseusCSProgramUserInput;
+
+        expect(userInput.status).toBe("incomplete");
+        expect(userInput.message).toBe(null);
+    });
+
+    it("should snapshot default user input before user interaction", () => {
+        // Arrange
+        const {renderer} = renderQuestion(question1);
+
+        // Act
+        const userInput = renderer.getUserInput();
+
+        // Assert
+        expect(userInput).toMatchInlineSnapshot(`
+            [
+              {
+                "message": null,
+                "status": "incomplete",
+              },
+            ]
+        `);
+    });
```

Comment:
@handeyeco So I added two different versions of what seem to be possibly the same test. Do either of these seem valuable? I wasn't able to figure out how to get an updated message from the renderer, but we can test the default at least.

Reply:
I think the first one is useful. Thanks!